backend/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "final-project",
```

Comment:
Instead of 'final-project' which sounds like generic name, we can use something which is meaningful and gives more context. Such as 'cs545-final-project' or 'clicknsplit-backend'